draft-ietf-quic-tls.md

```diff
@@ -1086,13 +1085,23 @@ before the final TLS handshake messages are received.  A client will be unable
 to decrypt 1-RTT packets from the server, whereas a server will be able to
 decrypt 1-RTT packets from the client.
 
-However, a server MUST NOT process data from incoming 1-RTT protected packets
-before verifying either the client Finished message or - in the case that the
-server has chosen to use a pre-shared key - the pre-shared key binder (see
-Section 4.2.11 of {{!TLS13}}).  Verifying these values provides the server with
-an assurance that the ClientHello has not been modified.  Packets protected with
-1-RTT keys MAY be stored and later decrypted and used once the handshake is
-complete.
+Until the handshake is considered complete, the connection and key exchange
+are not properly authenticated at the server.  Even though 1-RTT keys are
+available to a server after receiving the first handshake messages from a
+client, the server cannot consider the client to be authenticated until its
+TLS stack has indicated the handshake to be complete.  Therefore, a server
```

Comment:
See previous comment about the conditions for authentication of a peer.  This confuses them a little.

Basically, with a PSK binder you can be assured of identity before the handshake is complete.

Reply:
Made changes to this effect